packages/front-end/styles/global.scss

```diff
@@ -569,16 +580,19 @@ pre {
   }
   .won {
     color: #09ac17;
+    font-weight: bold;
     &.hover {
       color: #11ce21;
     }
+
     &.results-ctw,
     &.results-pval {
       background: linear-gradient(-40deg, #09ac1730, #09ac1716);
     }
   }
   .lost {
     color: #ce3729;
+    font-weight: bold;
     &.hover {
       color: #ee1500;
     }
```

Comment:
minor: offset bug that only shows up for thinker axis that we can fix on line 619:
```
  .visx-axis-left .visx-axis-line {
    // axis line has a 0.5 pixel offset. reverse it
    transform: translate(0.5px, -0.5px);
  }
```